torchgeo/datasets/so2sat.py

```diff
@@ -150,6 +183,7 @@ def __init__(
 
         Args:
             root: root directory where dataset can be found
+            version: one of "2", "3_random", "3_block", or "3_culture_10"
```

Comment:
Can you add a description of what `3_*` means? Either in the class or `__init__` docstring.